assignments/objects.js

```diff
@@ -19,26 +19,74 @@ const example = {
 
 // Write your intern objects here:
 
+const employee1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const employee2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@inypic.com",
+  "gender": "M",
+  "speak": "Hello, my name is Kennan"
+}
+
+const employee3 = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@illinois.edu",
+  "gender": "M"
+}
+
+const employee4 = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "qmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const employee5 = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  "multiplyNums": function (num1, num2) {
+    let sum = num1 * num2
+    return sum
+  }
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(employee1.name)
 
 // Kennan's ID
+console.log(employee2.id)
 
 // Keven's email
+console.log(employee3.email)
 
 // Gannie's name
+console.log(employee4.name)
 
 // Antonietta's Gender
+console.log(employee5.gender)
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+console.log(employee2.speak)
```

Comment:
then you could call your method by invoking it in the console log:

console.log(kennan.speak());